client/signup/config/flows-pure.js

```diff
@@ -258,7 +258,7 @@ export function generateFlows( {
 		};
 
 		flows[ 'ecommerce-onboarding' ] = {
-			steps: [ 'user', 'site-type', 'domains', 'plans' ],
+			steps: [ 'user', 'site-type', 'domains', 'plans-ecommerce' ],
```

Comment:
I think we need to replicate this change in the `ecommerce` flow above, and possibly the `store-woo` flow below, as well.

Reply:
On second thought, this change is only relevant when we have the information that the user wants to create a store, which comes from the `site-type` step. So this change is only relevant to the `ecommerce-onboarding` flow. We're good as is.